Neos.ContentRepository/Classes/Security/Authorization/Privilege/Node/NodePrivilegeContext.php

```diff
@@ -104,6 +105,32 @@ public function isDescendantNodeOf($nodePathOrIdentifier)
         return substr($this->node->getPath() . '/', 0, strlen($nodePath)) === $nodePath;
     }
 
+    /**
+     *
+     * @param string|array $nodeTypes A single or an array of fully qualified NodeType name(s), e.g. "Neos.Neos:Document"
+     * @return boolean true if the given node matches otherwise false
+     */
+    public function isDescendantOfNodetype($nodeTypes)
```

Comment:
Naming suggestion: isDescendantOfType (we have other methods in here where we abbreviate NodeType with "type" in this class)